File: Patterns/js/pattern9.js
/*

n = 4

A B C D
B C D E
C D E F
D E F G

*/

function pattern(n) {
  let ans = "";
  let row = 1;
  let startingChar = 65;
  while (row <= n) {
    let col = 1;
    let val = startingChar;
    while (col <= n) {
      ans += String.fromCharCode(val) + " ";
      val++;
      col++;
    }
    startingChar++;
    row++;
    ans += "\n";
  }

  console.log(ans);
}

pattern(4);
pattern(10);

File: Patterns/go/pattern9.go
package main

import "fmt"

/*

n = 4

A B C D
A B C D
A B C D
A B C D

*/


func main() {
	var n int
	fmt.Scan(&n)
	
	row := 1
	for row <= n {
		col := 1
		value := 'A'
		for col <= n {
			fmt.Print(string(value), " ")
			value++
			col++
		}
		fmt.Print("\n")
		row++
	}
}